src/main/java/com/amazonaws/services/kinesis/log4j/KinesisAppender.java

```diff
@@ -70,6 +70,49 @@ private void error(String message, Exception e) {
     throw new IllegalStateException(message, e);
   }
 
+    /**
```

Comment:
Apologies for not getting back to this earlier. Can you please add documentation on these new settings on the README.md file as well? I will commit this pull request as soon as you update the documentation. We appreciate your contribution and look forward to more in the future.